app/models/reaction.rb

```diff
@@ -101,7 +101,7 @@ def bust_reactable_cache_without_delay
   end
 
   def update_reactable_without_delay
-    Reactions::UpdateReactableJob.perform_now(id)
+    Reactions::UpdateReactableWorker.perform_async(id)
```

Comment:
This needs to be `Reactions::UpdateReactableWorker.new.perform(id)`

Reply:
Ok @mstruve. Doing the changes.